### DISCUSSION TITLE
Milestones question

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#16393] [General]
Hi guys, 
1. it seems like 6.4 is released to the public but in milestones I can see 95% complete. How to understand it?
2. yetiforce 7.0 - does developer planning some sort of radical GUI changes? jump from 6.4 to 7.0 is pretty wide :)

Best

**Comment by mariuszkrzaczkowski** [comment]
> 1. it seems like 6.4 is released to the public but in milestones I can see 95% complete. How to understand it?

Corrected
> 2. yetiforce 7.0 - does developer planning some sort of radical GUI changes? jump from 6.4 to 7.0 is pretty wide :)

We are not planning major changes to the GUI in this release
